docs/release-notes/cloud/cloud-2023-12-DD-gh-admin-permission-not-requested.md

```diff
@@ -0,0 +1,60 @@
+
+!!! info "This release note applies only to GitHub"
+
+For increased security, Codacy is deprecating the usage of **repository SSH keys** for Git operations on GitHub in favour of **installation access tokens**.
+
+This change translates into an important security improvement by reducing the liability of Codacy GitHub App, as it no longer requests [read and write repository permissions for Administration](https://docs.github.com/en/enterprise-cloud@latest/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration).
+
+## What do you need to do if you have a GitHub organization? {: id="to-do"}
+
+Make sure an organization owner [approves Codacy GitHub App updated permissions](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization, if not done yet.
+
+To use installation access tokens, Codacy GitHub App requires [repository read permissions for Contents](https://docs.github.com/en/enterprise-cloud@latest/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents). Thus, organization owners will be notified to review a request for this additional permission:
+
+![Codacy GitHub App updated permissions request](../images/2023-12-DD-gh-updated-permissions.png)
+
+If you have any questions or need help, please contact <mailto:support@codacy.com>.
+
+## Removal of repository permissions for Administration
+
+To ensure the [Codacy GitHub App updated permissions are approved](#to-do) on your GitHub organization before the removal of repository permissions for Administration, Codacy will execute a phased rollout according to the timeline below:
+
+<table>
+  <thead>
+    <th>Date</th>
+    <th>Event</th>
+    <th>What to expect</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>December DD, 2023</td>
+      <td>Codacy started using installation access tokens to clone and integrate with your repositories, and deprecated repositories SSH keys usage</td>
+      <td>
+        <p>From this day on, repositories SSH keys are used exclusively as a fallback mechanism when the Contents permission is missing.</p>
+        <p>If not done yet, make sure an organization owner <a href="#to-do">approves Codacy GitHub App updated permissions</a> on your GitHub organization.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>January DD, 2024</td>
+      <td>Brownout of repository permissions for Administration</td>
+      <td>
+        <p>On this day, Codacy won't use any fallback mechanism if the Contents permission is missing.</p>
+        <p>This will help you confirm that you already approved the updated permissions for Codacy GitHub App.<br/>If on this date your Codacy quality analysis fail or you have problems adding new repositories to Codacy, make sure an organization owner <a href="#to-do">approves Codacy GitHub App updated permissions</a> on your GitHub organization.</p>
```

Comment:
This one is tricky. We should say either `your Codacy quality analysis fails` (singular) or `your Codacy quality analyses fail` (plural), although "analyses" is unusual.

On a side note, another nitpick to add "the":

```suggestion
        <p>This will help you confirm that you already approved the updated permissions for the Codacy GitHub App.<br/>If on this date your Codacy quality analysis fails or you have problems adding new repositories to Codacy, make sure an organization owner <a href="#to-do">approves the updated permissions for the Codacy GitHub App</a> on your GitHub organization.</p>
```

Reply:
> This one is tricky. We should say either your Codacy quality analysis fails (singular) or your Codacy quality analyses fail (plural), although "analyses" is unusual.

Grammar typo here. 😄 I meant "Codacy quality analysis **fails**" (singular).

Tweaking your version to consider the role that needs to approve the updated permissions. 

 ```suggestion
        <p>This will help you confirm that the updated permissions for the Codacy GitHub App were already approved for your organization.<br/>If on this date your Codacy quality analysis fails or you have problems adding new repositories to Codacy, make sure an organization owner <a href="#to-do">approves the updated permissions for the Codacy GitHub App</a> on your GitHub organization.</p>
```